typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -520,35 +509,51 @@ class ContextFunder {
   }
 
   private async attemptToClaimFromIgp(chain: ChainName) {
-    const igpClaimThresholdEther = igpClaimThresholdPerChain[chain];
+    const igpClaimThresholdEther =
+      this.igpClaimThresholdPerChain[chain] ||
```

Comment:
Will this `igpClaimThresholdPerChain` include the hardcoded config if none is passed as an arg?

Reply:
not directly - what happens is that we read the config for a given environment and then run a keyfunder with the respective command line args based on the config

when we deploy/update keyfunder in k8s, what happens is we update the args configured in the cron job

so `igpClaimThresholdPerChain` here should only include the chains we pass in, to avoid being hardcoded to a specific environment

Reply:
I ask because where possible, I think it would be good to fall back on pre configured defaults like
```
 igpClaimThresholdPerChain: {
    arbitrum: '0.1',
    ancient8: '0.1',
    avalanche: '2',
    base: '0.1',
    blast: '0.1',
    bsc: '0.3',
    celo: '5',
    ethereum: '0.2',
    fraxtal: '0.1',
    gnosis: '5',
    inevm: '3',
    linea: '0.1',
    mantapacific: '0.1',
    mode: '0.1',
    moonbeam: '5',
    optimism: '0.1',
    polygon: '20',
    polygonzkevm: '0.1',
    redstone: '0.1',
    scroll: '0.1',
    sei: '2',
    viction: '2',
    zetachain: '20',
  },
```

instead of a 5th of the desired balance. Is that what happens in this change?

Reply:
discussed offline - explained how keyfunder config related to args